kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/CsrServiceImpl.java

```diff
@@ -281,6 +281,7 @@ public void processCallSummaryRecord(MotechEvent event) {
 
         CallRetry callRetry = callRetryDataService.findBySubscriptionId(subscriptionId);
         Subscription subscription = subscriptionDataService.findBySubscriptionId(subscriptionId);
+        resetWelcomeFlagInSubscription(subscription);
```

Comment:
Discussed with @frankhuster .
We agreed to reset in all cases.

But now i am not sure if call drops in between welcome message, so are we able to play welcome message again or not ?